### DISCUSSION TITLE
Do we not have a sleep timer?

### DISCUSSION THREAD

**Discussion by krishnarajt** [namidaco/namida#401] [General]
If we do I can't find it, can someone please let me know where it is? 

For an app with every possible configuration, a sleep timer seems like one of the things to check off in the first iteration, how do we not have it / not have it be instantly visible to users?

**Comment by MSOB7YY** [comment]
yes it's right in the drawer!
![Screenshot_20241117-123142_Namida](https://github.com/user-attachments/assets/38cfba16-3b29-4e20-9bf3-61c4750cbf67)

**Reply by krishnarajt** [reply]
Ohh! Thank you so much!
